src/components/CurrentPositionMarker.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import '../App.css';
+
+interface Coordinates {
+  lat: number;
+  lng: number;
+}
```

Comment:
You could use `GeoLocation` exported in ClinicFinder

Reply:
Wait, are they actually used?

Reply:
no ur right